README.md

```diff
@@ -16,7 +16,7 @@
   <br>
   <a href="https://github.com/neo-project/neo"><strong>Neo</strong></a>
   ·
-  <a href="https://github.com/neo-project/neo-vm">Neo VM</a>
+  <a href="https://github.com/neo-project/neo/tree/master/src/Neo.VM">Neo VM</a>
```

Comment:
Maybe we no longer need it? It's intended to unify various repositories IIUC, but it's the same repo now. We don't have Neo.Json here, for example.

Reply:
But neovm will still be published as a standalone packege right?

Reply:
So is Neo.Json.